app/experimenter/experiments/migrations/0163_auto_20210325_2041.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.1.7 on 2021-03-25 20:41
+
+from django.db import migrations, models
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+def update_changelogs(apps, schema_editor):
+    NimbusChangeLog = apps.get_model("experiments", "NimbusChangeLog")
+    db_alias = schema_editor.connection.alias
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="REVIEW").update(
+        old_status=NimbusConstants.Status.DRAFT.name,
+        old_publish_status=NimbusConstants.PublishStatus.REVIEW.name,
```

Comment:
The values are also wrong here.  `.name` is what is displayed to users, `.value` is what is stored in the db.